Dockerfile

```diff
@@ -1,26 +1,57 @@
-FROM ubuntu:20.04
+# build ktranslate
+FROM golang:1.17 as build
+RUN apt-get update && apt-get install -y libpcap-dev
+COPY . /src
+WORKDIR /src
+RUN make
+
+# maxmind dbs
+FROM alpine:latest as maxmind
+ARG MAXMIND_LICENSE_KEY
+RUN apk add -U curl tar
+ENV GEOLITE2_COUNTRY_FILE=GeoLite2-Country.mmdb
+ENV GEOLITE2_ASN_FILE=GeoLite2-ASN.mmdb
+RUN if [ -z "${MAXMIND_LICENSE_KEY}" ]; then echo "MAXMIND_LICENSE_KEY" not set; exit 1; fi
+RUN curl -o /tmp/country.tar.gz "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${MAXMIND_LICENSE_KEY}&suffix=tar.gz" && \
+	tar zxf /tmp/country.tar.gz --strip-components 1 -C /
+RUN curl -o /tmp/asn.tar.gz "https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-ASN&license_key=${MAXMIND_LICENSE_KEY}&suffix=tar.gz" && \
+	tar zxf /tmp/asn.tar.gz --strip-components 1 -C /
+
+# snmp profiles
+FROM alpine:latest as snmp
+RUN apk add -U git
+RUN git clone https://github.com/kentik/snmp-profiles /snmp
 
+# main image
+FROM ubuntu:20.04
```

Comment:
Unrelated, but I've been thinking about moving to alpine from ubuntu to reduce the vulnerability scan count. Thoughts?

Reply:
I'm always happy to move to Alpine :). I'll switch and see if there are any issues.